androidApp/src/main/java/com/mbta/tid/mbta_app/android/MainApplication.kt

```diff
@@ -17,6 +18,14 @@ class MainApplication : Application() {
 
     override fun onCreate() {
         super.onCreate()
-        initKoin(appVariant, makeNativeModule(appCheck, socket.wrapped()), this)
+        initKoin(
+            appVariant,
+            makeNativeModule(
+                AccessibilityStatusRepository(applicationContext),
```

Comment:
praise: I like the way you approached putting the AccessibilityStatus in shared code